app/views/api/v2/analytics/user_analytics/_gender_filter.html.erb

```diff
@@ -1,13 +1,19 @@
 <form>
   <select class="card-dropdown <%= data_table_name %>" onchange='filterDataByGender("<%= data_table_name %>")'>
-    <% hypertension_option_text = statistics.dig(:metadata, :is_diabetes_enabled) ? 'Hypertension: ' : '' %>
+    <% if is_diabetes_enabled %>
+      <option value="all">
+        All
+      </option>
```

Comment:
Let's single line this for consistency with the other options. Makes it easier to parse.